pages/firststeps.md

```diff
@@ -39,11 +39,7 @@ Go to [this GitHub address](https://github.com/dogi/ole--vagrant-vi) to read thr
 
 ## Step 2 - Vagrant Tutorial
 
-#### Windows 
-Review these [Vagrant instructions](vagrantwin.md) to get familiar with Vagrant, since you will need to use it during your internship.
-
-#### MacOS(X) and Ubuntu
-Review these [Vagrant instructions](vagrant.md) to ensure that you have fully completed the previous step.
+Review these [Vagrant instructions](vagrant.md) to ensure that you have fully completed the previous step. You should be familiar with this since you will need to use it during your internship.
```

Comment:
Better to put this sentence `You should be familiar with this since you will need to use it during your internship.` in between heading 2 and heading 3 or in other words below `## Step 2 - Vagrant Tutorial`